src/chat_do.py

```
@@ -0,0 +1,406 @@
+            await self.env.DB.prepare(
+                "INSERT INTO chat_message (id, classroom_id, user_id, display_name, content, created_at) VALUES (?, ?, ?, ?, ?, datetime('now'))"
+            ).bind(msg_id, classroom_id, user_id, display_name, stored_content).run()
+            return True
+        except Exception as exc:
+            if "no such table" in str(exc).lower():
+                await self._ensure_table()
+                try:
+                    await self.env.DB.prepare(
+                        "INSERT INTO chat_message (id, classroom_id, user_id, display_name, content, created_at) VALUES (?, ?, ?, ?, ?, datetime('now'))"
+                    ).bind(msg_id, classroom_id, user_id, display_name, stored_content).run()
+                    return True
```

Ruff (0.16.0)

[warning] Consider moving this statement to an `else` block

(TRY300)